### PR TITLE
[Security] Fix composer.json

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #16434 |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/Security/composer.json [hunk 1]
```diff
@@ -39,11 +39,6 @@
         "ircmaxell/password-compat": "~1.0",
         "symfony/expression-language": "~2.6"
     },
-    "replace": {
-      "symfony/security-acl": "self.version",
```
Comment (WouterJ) [inline src/Symfony/Component/Security/composer.json]: actually, only this entry should be removed. Core and HTTP should still be replaced by the current version (as it's included in the Security package).

Comment (nicolas-grekas) [inline src/Symfony/Component/Security/composer.json]: these lines are duplicates: a replace section already exists a few lines above in the file

**Comment by stof** [comment]
@nicolas-grekas this should be reverted (and btw, Symfony 2.7 still replaces security-acl, it has been removed only in 2.8)

**Comment by nicolas-grekas** [comment]
See https://github.com/symfony/symfony/pull/16437/files#diff-303420d112ba190b0f9edb9c9806d7b2R24

**Pull request merged**